### ISSUE TITLE
RDCYCLE definition

### ISSUE THREAD

**Issue by ilg-ul** [riscv/riscv-isa-manual#140]
The current definition in rv32.tex:1171 reads:

> The RDCYCLE pseudoinstruction reads the low XLEN bits of the {\tt
  cycle} CSR which holds a count of the number of clock cycles
**executed by the processor core** on which the hart is running from
an arbitrary start time in the past. 

Given that a core may support multiple harts (1.1  "... A RISC-V- compatible core might support multiple RISC-V-compatible hardware threads, or harts, through multithreading."), is the RDCYCLE definition as the "number of clock cycles executed by the processor core" correct? 

Or it should read "number of clock cycle executed by the current hart", to account for each hart going to sleep differently and possibly not counting cycles at all in deep sleep?

In other words, is the cycle counter unique to the core, or specific to a hart?

**Comment by ilg-ul** [comment]
> ... It is per-core

ok, thank you.
